src/design-system/color/AccentColorContext.docs.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box } from '../components/Box/Box';
+import { Text } from '../components/Text/Text';
+import * as Docs from '../docs/components';
+import { Docs as DocsType } from '../docs/types';
+import source from '../docs/utils/source.macro';
+import { AccentColorProvider } from './AccentColorContext';
+
+const docs: DocsType = {
+  category: 'Color',
+  description: [
+    <Docs.Text key="">
```

Comment:
All description nodes have the same key which is causing React to throw errors. Maybe this should be a fragment which the Stack component can then flatten?

Reply:
Good idea!!